libs/net/services/Runners/BaseService.cs

```diff
@@ -49,6 +50,7 @@ public BaseService(string[] args)
         this.Configuration = Configure(builder).Build();
         ConfigureServices(builder.Services);
         this.App = builder.Build();
+        Console.OutputEncoding = Encoding.UTF8;
```

Comment:
Trying to fix the log encoding issue in Openshift.  Not sure how to do it however.